### PR TITLE
Update welcome.txt

### PR DESCRIPTION
**Pull request by raynamharris**
made a few updates including some formatting and a place to add optional details.

cc @davis68 @gvwilson @ChristinaLK

### CODE REVIEW

File: files/messages/welcome.txt [hunk 1]
```diff
@@ -1,20 +1,34 @@
...
+2.  Please go to the Software Carpentry lessons page (http://software-carpentry.org/lessons/) and the Data Carpentry lessons page (http://www.datacarpentry.org/lessons/), have a look at what we currently teach, and then choose *one episode* from the list at the bottom of this message and read through it carefully. You will be using your selected episode for several in-class exercises, so be sure you are comfortable with the content.
+
+**What to bring to the workshop:**
+
+1.  We will be recording one another teaching in pairs or threes during the class, so please bring a cell phone or hand-held camera that can record audio and video. It doesn't have to be high-quality, but it should be good enough that you can understand what someone is saying.
+
+2.  Please also bring a laptop powerful enough to teach on so that you can take part in all of the practical exercises (i.e. SQL, Git, Python, R, or Unix) and a power cable. You will need this laptop on both day of the instructor training.
```
Comment (lexnederbragt) [inline files/messages/welcome.txt]: This makes it sound like we will do a lot of exercises in all these languages. Perhaps drop the list of languages and rephrase to make clear that participant is expected to teach something from his chosen lesson?

File: files/messages/welcome.txt [hunk 2]
```diff
@@ -1,20 +1,34 @@
-Hello everyone, and thank you for signing up for our upcoming instructor training session. In order to prepare:
+Hello everyone,
+
+Thank you for signing up for our upcoming instructor training workshop. 
+
+Please this website for specific details about the workshop [INSERT WORKSHOP WEBSITE LINK HERE].
```
Comment (lexnederbragt) [inline files/messages/welcome.txt]: Missing a verb :-) 'Please see this website'

File: files/messages/welcome.txt (hunk 1 above)
Comment (raynamharris) [inline files/messages/welcome.txt]: Good point.  Let me remove list of languages.

File: files/messages/welcome.txt [hunk 3]
```diff
@@ -1,20 +1,34 @@
...
+2.  Please go to the Software Carpentry lessons page (http://software-carpentry.org/lessons/) and the Data Carpentry lessons page (http://www.datacarpentry.org/lessons/), have a look at what we currently teach, and then choose *one episode* from the list at the bottom of this message and read through it carefully. You will be using your selected episode for several in-class exercises, so be sure you are comfortable with the content.
+
+**What to bring to the workshop:**
+
+1.  We will be recording one another teaching in pairs or threes during the class, so please bring a cell phone or hand-held camera that can record audio and video. It doesn't have to be high-quality, but it should be good enough that you can understand what someone is saying.
+
+2.  Please also bring a laptop powerful enough to teach on so that you can take part in all of the practical exercises and a power cable. You will need this laptop on both day of the instructor training.
```
Comment (lexnederbragt) [inline files/messages/welcome.txt]: 'and a power cable' --> ', remember to bring the charger also.'

**Pull request merged**